docs/5_1_scheduled_execution_math.adoc

```diff
@@ -5,25 +5,12 @@
 In Scheduled Execution the importer has to ensure that model partitions are scheduled according to their activation time and priorities.
 This is realized in <<ClockActivationMode>> by calling <<fmi3ActivateModelPartition>> triggering their respective Clocks.
 
-After <<fmi3ActivateModelPartition>> has been called for a <<fixed-clock,`fixed`>>, <<tunable-clock,`tunable`>> or <<changing-aperiodic-clock,aperiodic>> Clock the FMU provides the information on when the Clock will tick again, i.e. when the corresponding model partition must be scheduled the next time.
```

Comment:
Why is this deleted?

Reply:
That is already described in the clocks section.